plugins/item_tasks/web_client/templates/selectTaskViewTargetDescription.pug

```diff
@@ -0,0 +1,3 @@
+.g-selected-task
+  h4.selected-task-name= task.name()
+  .selected-task-description= task.get('description')
```

Comment:
I'm not sure what the value of adding the description here again is. We can already see the description above, on the `PaginateTasksWidget` above this.